tests/UriTest.php

```diff
@@ -692,6 +692,15 @@ public function testInternationalizedDomainName()
         $uri = new Uri('https://яндекAс.рф');
         self::assertSame('яндекaс.рф', $uri->getHost());
     }
+
+    public function testIPv6Host()
+    {
+        $uri = new Uri('https://[2a00:f48:1008::212:183:10]');
+        self::assertSame('[2a00:f48:1008::212:183:10]', $uri->getHost());
+
+        $uri = new Uri('http://[2a00:f48:1008::212:183:10]');
+        self::assertSame('[2a00:f48:1008::212:183:10]', $uri->getHost());
```

Comment:
We dont need these. They are the same.

Reply:
Likely, though I'm rather on the safe side with small but common differences (http vs http*s*), especially when using custom parsing logic.